<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MagLev</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Port COM:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="COMBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Odśwież</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Połącz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Połącz z RTDAC:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rtdacButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Połącz</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nucleo</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RTDAC</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Eksperyment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="LED" name="led" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="LED" name="led_2" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="LED" name="led_3" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pasek postępu: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="experimentProgress">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuEksperyment">
    <property name="title">
     <string>Eksperyment</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionPlots"/>
    <addaction name="actionSaveData"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuApliacja">
    <property name="title">
     <string>Aplikacja</string>
    </property>
    <addaction name="actionTerminal"/>
    <addaction name="actionRTDAC"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuApliacja"/>
   <addaction name="menuEksperyment"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTerminal"/>
   <addaction name="actionRTDAC"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionPlots"/>
   <addaction name="actionSaveData"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/info_icon.png</normaloff>:/new/ikony/info_icon.png</iconset>
   </property>
   <property name="text">
    <string>Informacje</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/settings_icon.png</normaloff>:/new/ikony/settings_icon.png</iconset>
   </property>
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="actionRTDAC">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/kcmpci.png</normaloff>:/new/ikony/kcmpci.png</iconset>
   </property>
   <property name="text">
    <string>RTDAC</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/quit_icon.png</normaloff>:/new/ikony/quit_icon.png</iconset>
   </property>
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/terminal_icon.png</normaloff>:/new/ikony/terminal_icon.png</iconset>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionPlots">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/plot_icon.png</normaloff>:/new/ikony/plot_icon.png</iconset>
   </property>
   <property name="text">
    <string>Wykresy</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/save_icon.png</normaloff>:/new/ikony/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz dane</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/start_icon.png</normaloff>:/new/ikony/start_icon.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="ikony.qrc">
     <normaloff>:/new/ikony/stop_icon.png</normaloff>:/new/ikony/stop_icon.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LED</class>
   <extends>QWidget</extends>
   <header>LED.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ikony.qrc"/>
 </resources>
 <connections/>
</ui>
